scripts/generateTS.sh

```diff
@@ -43,9 +43,11 @@ replace_string_in_file() {
   # and replaces them with:
   #   "$ref": "./
   local old_string="\"\$ref\":\ \"https:\/\/api.dataplatform.ibm.com\/schemas\/common-pipeline\/"$url"\/"
+  local old_string_with_common_canvas_path="\"\$ref\":\ \"https:\/\/api.dataplatform.ibm.com\/schemas\/common-canvas\/"$url"\/"
```

Comment:
Can we rename these to be consistent. So instead of old_string and old_string_with_common_canvas_path lets go with `common_pipeline_path` and `common_canvas_path`